lighthouse-core/gather/gatherers/dobetterweb/links-blocking-first-paint.js

```diff
@@ -21,17 +21,19 @@ const Gatherer = require('../gatherer');
 /* global document,window */
 
 /* istanbul ignore next */
-function collectLinksThatBlockFirstPaint() {
+function collectLinksThatBlockFirstPaint(target, matchMedia) {
```

Comment:
Maybe add a note that these params, while not used int his audit, were added for testability? I was confused why they were being added at first and we'll probably be confused looking back later.